apps/backlite/settings.js

```diff
@@ -1,19 +1,20 @@
 (function(back) {
   var FILE = "BackLite.settings.json";
-  // Load settings
-  var settings = Object.assign({
-      brightness: 0.3,
-  }, require('Storage').readJSON(FILE, true) || {});
+
+  // Load settings safely (avoid crashes on bad/missing JSON)
```

Comment:
Do you have some setup to reproduce the crash? I'd like to see what's going on there

Reply:
Ah makes sense, thanks for the details